recipe/install.py

```diff
@@ -0,0 +1,188 @@
+# binary, for any purpose, commercial or non-commercial, and by any
+# means.
+#
+# In jurisdictions that recognize copyright laws, the author or authors
+# of this software dedicate any and all copyright interest in the
+# software to the public domain. We make this dedication for the benefit
+# of the public at large and to the detriment of our heirs and
+# successors. We intend this dedication to be an overt act of
+# relinquishment in perpetuity of all present and future rights to this
+# software under copyright law.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+# MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+# IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+# For more information, please refer to <https://unlicense.org>
+
+"""A python install script for conda-build recipes with multiple outputs
+
+Used in outputs/script to split the files of a top-level package into multiple
+outputs instead of using the outputs/files dictionary of globs. The advantage
+of this approach is that you don't need to choose glob expressions that include
+files installed by this package while excluding files installed by the host
+dependencies. For example, this script just globs "include", whereas using
+outputs/files, you would need to glob "include/foo.h", "include/foo" and
+possibly others in order to exclude the headers from dependencies.
+
+To use this script, set your install prefix in your build script to a new
+directory named `$SRC_DIR / stage`. Change `basename` to the basename of the
+package.
+
+https://gist.github.com/carterbox/188ac74647e703cfa6700b58b076d712
+"""
+
+import glob
+import os
+import pathlib
+import re
+import shutil
+import typing
+import itertools
+
+target_platform = os.environ["target_platform"]
+STAGE = pathlib.Path(os.environ["SRC_DIR"]) / "stage"
+PREFIX = pathlib.Path(os.environ["PREFIX"])
+if target_platform[:3] == "win":
+    PREFIX = PREFIX / "Library"
+
+
+def glob_install(
+    include: typing.List[str],
+    exclude: typing.List[str] = [],
+):
+    included = set(
+        itertools.chain(
+            *(
+                glob.glob(
+                    str(STAGE / pathlib.Path(item)),
+                    recursive=True,
+                )
+                for item in include
+            )
+        )
+    )
+    excluded = set(
+        itertools.chain(
+            *(
+                glob.glob(
+                    str(STAGE / pathlib.Path(item)),
+                    recursive=True,
+                )
+                for item in exclude
+            )
+        )
+    )
+    for match in sorted(included - excluded):
+        match = pathlib.Path(match)
+        if match.exists():
```

Comment:
```suggestion
        if match.is_dir():
            continue
        if match.exists():
            # runtime error?
```

Reply:
I can't recall now why glob was returning paths that did not exist. I think a summary of the things that I want to install (copy) are files and symlinks to files (even if broken). I don't want to copy directories.

Reply:
I have updated the script to explicitly check for file or symlink.